packages/ckeditor5-image/docs/features/image.md

```diff
@@ -86,12 +87,15 @@ See the {@link features/image-upload Image upload} guide.
 
 Besides the ability to insert images by uploading them directly from your disk or via CKFinder, you can also configure CKEditor 5 to allow inserting images via source URL.
 
-In order to enable this option, configure {@link module:image/imageupload~ImageUploadPanelConfig#items `image.upload.panel.items`} like below:
+In order to enable this option, install the `ImageInsert` plugin and configure {@link module:image/imageupload~ImageUploadPanelConfig#items `image.upload.panel.items`} like below:
```

Comment:
As you already mentioned in https://github.com/ckeditor/ckeditor5/issues/7890#issuecomment-685703086 there's a natural followup to fix the config option name. I'll report a ticket for that.

Reply:
Done: #8034